### PR TITLE
Javascript

### PR DESCRIPTION
**Pull request by codingoak**

### CODE REVIEW

File: create.html [hunk 1]
```diff
@@ -36,6 +37,7 @@ <h2 class="create__headings">Add an answer</h2>
         <section>
           <h2 class="create__headings">Add tags</h2>
           <textarea
+            class="mb-20"
```
Comment (ttaannjjaa) [inline create.html:44]: Hi Daniel, i am not completely sure, but i had made a written notice during the lesson that `textarea` also needs an additional label tag for accessibility as it is a multiline input field

File: styles/components/card.css [hunk 2]
```diff
@@ -2,6 +2,12 @@
   padding: 30px;
   border: 1px solid black;
   position: relative;
+  display: grid;
```
Comment (ttaannjjaa) [inline styles/components/card.css:5]: very interesting idea - i like it 😁

**Review by ttaannjjaa — CHANGES_REQUESTED** [review]
Awesome work 😃

File: create.html (hunk 1 above)
Comment (codingoak) [reply create.html:44]: Yes you're totally right. I changed it and pushed the fixed solution. thx for watching

**Pull request merged**